app/templates/sectionsummary.html

```diff
@@ -0,0 +1,37 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% extends theme('section_confirmation.html') -%}
+
+{% block page_title -%}{{_("Section-Summary")}} - {{survey_title}}{% endblock -%}
+
+{% block sidebar %}
+  {% if navigation %}
+    {% include theme('partials/navigation.html') %}
+  {% endif %}
+{% endblock %}
+
+{% block form_content %}
+{% endblock %}
+
+{% block main -%}
+
+<div>
+    {% set group = content.summary.groups %}
+    <h1 class="summary__title saturn" id="{{group.id}}">{{content.summary.title}}</h1>
+    <div class="print__hidden panel panel--simple panel--success">
+        <strong>This section is now complete</strong>
+        <p>You can review your answers below</p>
+    </div>
+</div>
```

Comment:
We have a lot of duplication here with the summary.html. These should be merged and a little logic just to show these text differences.